scrolldelegator/depth.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileoverview Calls a function whenever a scroll depth is reached.
+ * Ratio-based depths (e.g. 25%, 50%, 75%) are supported.
+ */
+
+var objects = require('../utils/objects');
+var scrolldelegator = require('.');
+
+
+var defaultConfig = {
+  ratios: [0.25, 0.50, 0.75, 0.95]
```

Comment:
We can later enhance this by adding support for when certain query selectors are reached, or elements, etc. That could make it easier to implement those "fixed position" / scrolljacking designs. But right now I don't have a sample use case for that, so only ratios is implemented.